### PR TITLE
update organizations

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-5407](https://jira.cms.gov/browse/MAT-5407)
(Optional) Related Tickets:

### Summary
update organizations

I have used few JS logic to manipulate array lists relating to organization.

1. For removing given organizations
deleteDataList=[list of only orgs] eg:['abc','bb']
actualDataList=[{name:orgname, oid:1.2},...]
const res = actualData.filter(data => !deleteDataList.includes(data.name));

2. For Adding urls
var res= withouturlsdata.map((tes)=>{
   return {...tes,"url": urltestdata.filter((url)=>url.name===tes.name).map((data)=>data.url)[0] }
})

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/cms/gov/madie/measure/services/MeasureService.java [hunk 1]
```diff
@@ -67,7 +67,8 @@ public Measure updateMeasure(
             measureUtil.validateAllMeasureGroupReturnTypes(updateElm(updatingMeasure, accessToken));
         // no errors were encountered so remove the ELM JSON error
         // TODO: remove this when backend validations for CQL/ELM are enhanced
```
Comment (gregory-akins) [inline src/main/java/cms/gov/madie/measure/services/MeasureService.java]: Is this comment still relevant?

File: src/main/java/cms/gov/madie/measure/config/AddOrganizationsChangeUnit.java [hunk 2]
```diff
@@ -22,9 +22,11 @@ public AddOrganizationsChangeUnit(ObjectMapper objectMapper) {
 
   @Execution
   public void addOrganizations(OrganizationRepository organizationRepository) throws IOException {
-    InputStream inputStream = getClass().getResourceAsStream("/data/organizations.json");
```
Comment (adongare) [inline src/main/java/cms/gov/madie/measure/config/AddOrganizationsChangeUnit.java:25]: can we delete organization.json? or is it needed?

Comment (sb-prateekkeerthi) [reply src/main/java/cms/gov/madie/measure/config/AddOrganizationsChangeUnit.java:25]: I wasn't sure if we would need that list in the future so I haven't deleted it. I can delete it

File: src/main/java/cms/gov/madie/measure/services/MeasureService.java (hunk 1 above)
Comment (sb-prateekkeerthi) [reply src/main/java/cms/gov/madie/measure/services/MeasureService.java]: I am not sure about this Greg, I haven't worked on it. This change is seen based on my checkstyle

**Review by adongare — APPROVED** [review]
